QUARKUS-6561.md

```diff
@@ -0,0 +1,30 @@
+# QUARKUS-6561 - OIDC Client filter - allow to trigger token refresh when REST client request results in 401
+
+JIRA: https://issues.redhat.com/browse/QUARKUS-6551
+
+Upstream PRs:
+* https://github.com/quarkusio/quarkus/pull/49003
+
+Documentation: https://quarkus.io/version/main/guides/security-openid-connect-client-reference#rest-client-oidc-filter
+
+## Scope of testing
+New tests will be added verifying that OIDC client will automatically get a new token, if configured so, using:
+* Configuration property `quarkus.rest-client-oidc-filter.refresh-on-unauthorized=true`
+  * Also with named client's configuration property.
+* Configured in code using `AbstractOidcClientRequestReactiveFilter::refreshOnUnauthorized()`
+* Client will not refresh token, if not configured to do so.
```

Comment:
You probably know that, but the client will refresh token if not configured. The only difference is that it will happen on fly when the expired token is detected, rather than beforehand.